### DISCUSSION TITLE
i2d_SM2_Ciphertext() function prototype code

### DISCUSSION THREAD

**Discussion by HUGO313L** [openssl/openssl#22468] [Q&A]
where is the function i2d_SM2_Ciphertext() prototype code，I can't find it.

**Comment by slontis** [comment]
It sits inside a macro i.e. See crypto.sm2/sm2_crypt.c
````
IMPLEMENT_ASN1_FUNCTIONS(SM2_Ciphertext)
````

**Comment by slontis** [comment]
NOTE: Loading c into an editor that allows you to find things is really handy. (e.g. eclipse using CTRL+SHIFT T)
